app/services/datetime_parser.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+class DatetimeParser
+  def initialize(date:, time:, issue_prefix:)
+    @raw_date = date.to_h
+    @raw_time = time.to_s
+    @issue_prefix = issue_prefix
+  end
+
+  def issues
+    Requirements::CheckerIssues.new(issue_items || [])
```

Comment:
I don't think copying and pasting the class from Content Publisher like this works. This class `Requirements::CheckerIssues` doesn't exist in Collections Publisher, neither does `Requirements::Issue`, so the tests must have failed when this code was added.